vmm/src/memory_manager.rs

```diff
@@ -64,7 +64,26 @@ struct HotPlugState {
     removing: bool,
 }
 
-pub type MemoryZones = HashMap<String, Vec<Arc<GuestRegionMmap>>>;
+#[derive(Default)]
+pub struct MemoryZone {
+    regions: Vec<Arc<GuestRegionMmap>>,
+    virtiomem_region: Option<Arc<GuestRegionMmap>>,
```

Comment:
I think s/virtiomem_region/virtio_mem_region/

It scans much better and is more consistent with other parts of the code.